src/views/preview/project-view.jsx

```diff
@@ -604,7 +604,7 @@ class Preview extends React.Component {
                             extensions={this.state.extensions}
                             faved={this.state.clientFaved}
                             favoriteCount={this.state.favoriteCount}
-                            isFullScreen={this.state.isFullScreen}
+                            isFullScreen={this.props.fullScreen}
```

Comment:
Note that this prop seems to have been pointing to a variable that no longer exists! I had to correct it so that `preview/presentation.jsx` would be able to use the right value.